R/ConceptSet.R

```diff
@@ -332,17 +332,11 @@ resolveConceptSetId <-
            formatName = FALSE) {
     .checkBaseUrl(baseUrl)
     
-    readCsv <- function(file) {
-      concepts <- read.csv(
-        file = substring(file, first = 3),
-        header = FALSE,
-        sep = ",",
-        strip.white = TRUE,
-        blank.lines.skip = TRUE,
-        skipNul = TRUE
-      )
-      names(concepts) <- as.character(apply(concepts[1, ], 1, paste))
-      concepts <- concepts[-1, ]
+    .readCsv <- function(file) {
+      concepts <- readr::read_csv(file = file)
```

Comment:
I recommend using `col_types = readr::cols()` when calling read_csv to prevent the warning message that will confuse users.

Also, depending on the type of data in the CSV file, you may want to specifiy `guess_max = 1e7`, because else only the first 1,000 rows (I think) are used to guess the column types.

Reply:
That works! Thank you for the suggestion